File: app/src/main/java/cg/lastfm/data/ArtistMatchesResults.java
package cg.lastfm.data;

import com.google.gson.annotations.SerializedName;

public class ArtistMatchesResults {
    @SerializedName("opensearch:Query")
    public OpenSearchQuery query;

    @SerializedName("opensearch:totalResults")
    public long totalResults;

    @SerializedName("opensearch:startIndex")
    public long startIndex;

    @SerializedName("opensearch:itemsPerPage")
    public long itemsPerPage;

    @SerializedName("artistmatches")
    public ArtistMatches artistMatches;
}

File: app/src/main/java/cg/lastfm/data/ArtistMatchesResults.kt
package cg.lastfm.data

import com.google.gson.annotations.SerializedName

data class ArtistMatchesResults(
        @SerializedName("opensearch:Query")
        val query: OpenSearchQuery?,

        @SerializedName("opensearch:totalResults")
        val totalResults: Long,

        @SerializedName("opensearch:startIndex")
        val startIndex: Long,

        @SerializedName("opensearch:itemsPerPage")
        val itemsPerPage: Long,

        @SerializedName("artistmatches")
        val artistMatches: ArtistMatches?
)
